src/main/java/it/gov/pagopa/payhub/activities/activity/VerifyAuthorizationInstallmentsActivity.java

```diff
@@ -0,0 +1,30 @@
+package it.gov.pagopa.payhub.activities.activity;
+
+import it.gov.pagopa.payhub.activities.dao.OrganizationInstallmentTypeDao;
+import it.gov.pagopa.payhub.activities.dto.InstallmentsOperatorDTO;
+import it.gov.pagopa.payhub.activities.dto.OrganizationInstallmentTypeDTO;
+import it.gov.pagopa.payhub.activities.exception.custom.ValidatorException;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class VerifyAuthorizationInstallmentsActivity {
+
+    private final OrganizationInstallmentTypeDao organizationInstallmentTypeDao;
+
+    public VerifyAuthorizationInstallmentsActivity(OrganizationInstallmentTypeDao organizationInstallmentTypeDao) {
+        this.organizationInstallmentTypeDao = organizationInstallmentTypeDao;
+    }
+
+    public OrganizationInstallmentTypeDTO verifyAuth(InstallmentsOperatorDTO installmentsOperatorDTO, String username, Long mygovEnteId){
```

Comment:
let's write some javadoc (to reference other docs, you could use as an example: @{see VerifyAuthorizationInstallmentsActivity} )

`It will verify if the operator is authorized on the DebtPositionType

@return ...
@throws OperatorNotAuthorizedException if the operator is not allowed on the DebtPositionType`

Reply:
I need it to map the entity EnteTipoDovuto